bin/historical/migrations/_common.py

```diff
@@ -0,0 +1,41 @@
+import os
+import subprocess
+import importlib
+import logging
+
+import emission.core.get_database as edb
+
+DB_HOST_TEMPLATE = "mongodb://localhost:27017/openpath_prod_REPLACEME"
+
+proc = subprocess.run(
+      'rm -rf nrel-openpath-deploy-configs && ' +
+      'git clone --no-checkout https://github.com/e-mission/nrel-openpath-deploy-configs.git && ' +
+      'cd nrel-openpath-deploy-configs && ' +
+      'git ls-tree -r --name-only HEAD | grep configs/',
+      shell=True,
+      capture_output=True,
+      text=True)
+filenames = proc.stdout.replace("configs/", "").split("\n")
+
```

Comment:
I assume this works, and I don't want the perfect to be the enemy of the good, but this seems overly convoluted. Why isn't this just

```suggestion
      'rm -rf nrel-openpath-deploy-configs && ' +
      'git clone https://github.com/e-mission/nrel-openpath-deploy-configs.git)'
filenames = os.listdir('nrel-openpath-deploy-configs')
```

It seems much cleaner than capturing stdout and mucking around with it.

Reply:
```suggestion
  with tempfile.TemporaryDirectory() as tmpdirname:
      print('created temporary directory', tmpdirname)
      proc = subprocess.run('git clone --no-checkout https://github.com/e-mission/nrel-openpath-deploy-configs.git ' + tmpdirname)
filenames = os.listdir(tmpdirnme+"/configs")
```

Reply:
I did this to avoid downloading the actual files since all we needed were the filenames. May have been overengineered

Reply:
I thought it may be something like that (no-checkout) but yes, I thought it was overly complex, and having the full configs might be helpful for some functions (e.g. maybe the function will be sensitive to the variation in the configs).

Reply:
Also fixed in https://github.com/e-mission/e-mission-server/pull/1009/commits/947492d1b7ec509e7afc0f7b6af0a9eca52ac93a